java/mtsj/api/src/main/java/com/devonfw/application/mtsj/usermanagement/common/api/User.java

```diff
@@ -8,6 +8,10 @@
 
   public void setUsername(String username);
 
+  public String getPassword();
```

Comment:
You re-introduced a severe security issue again with this change:
https://github.com/devonfw/my-thai-star/issues/55
Please rework and prevent exposing the password again.

Reply:
Done